app/views/hyrax/dashboard/collections/edit.html.erb

```diff
@@ -4,33 +4,9 @@
   <h1><span class="fa fa-edit"></span><%= t('.header', type_title: @collection.collection_type.title, title: @form.title.first) %></h1>
 <% end %>
 
-<% unless has_collection_search_parameters? %>
-  <div class="row">
-    <div class="col-md-12">
-      <%= render 'flash_msg' %>
-      <%= render 'form' %>
-    </div>
+<div class="row">
+  <div class="col-md-12">
+    <%= render 'flash_msg' %>
+    <%= render 'form' %>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <div class="panel panel-default">
-        <div class="panel-body">
-          <%= render 'edit_actions' %>
-
-          <div class="row m-t-3">
-            <h2 class="col-xs-12 col-sm-6"><%= t('hyrax.collection.edit.manage_items') %></h2>
-            <div class="col-xs-12 col-sm-6">
-              <%= render 'hyrax/collections/search_form', presenter: @collection %>
-            </div>
-          </div>
-          <%= render 'hyrax/my/did_you_mean' %>
-          <%= render 'hyrax/my/facet_selected' %>
-          <%= render 'sort_and_per_page', collection: @collection if @response.response['numFound'] > 0 %>
-          <%= render 'document_list', documents: @member_docs, document_list_format: "dashboard" %>
```

Comment:
Can `app/views/hyrax/collections/_document_list.html.erb` be removed now?

Reply:
Same answer as above.

Reply:
and, sorry, I meant `app/views/hyrax/dashboard/collections/_document_list.html.erb`